javascript/packages/eslint-config-nebenan/package.json

```diff
@@ -6,7 +6,7 @@
   "readmeFilename": "README.md",
   "repository": "goodhood-eu/styleguide",
   "bugs": "https://github.com/goodhood-eu/styleguide/issues",
-  "version": "4.0.2",
+  "version": "5.0.0",
```

Comment:
Not sure if it's a breaking release. We pretty much 'add' support for ?svg urls. All old code should go through just fine?

Reply:
hm... yes, i think you are right